compiler/damlc/daml-stdlib-src/DA/Internal/Interface.daml

```diff
@@ -78,6 +73,11 @@ class HasFromInterface t i where
   -- the interface value `value` into the template type `MyTemplate`.
   fromInterface : i -> Optional t
 
+  -- | HIDE
+  -- Like 'fromInterface', but throws an error if the underlying template type
+  -- of the interface value doesn't match the type of the provided ContractId.
```

Comment:
Because the contract id isn't the desired type anymore, and because this raises a fatal error instead of an exception:

```suggestion
  -- Like 'fromInterface', but raises a fatal error (`WronglyTypedContract`) if the
  -- underlying template type of the interface value doesn't match the desired type.
```

Reply:
ah good catch, thanks!